src/elements/hx-search-control/index.js

```diff
@@ -13,21 +14,20 @@ export class HXSearchControlElement extends HXFormControlElement {
         return 'hx-search-control';
     }
 
-    $onCreate () {
```

Comment:
Was there an error that caused this to be removed?

Reply:
Yes James, it throws an error while running yarn test "Error: Uncaught TypeError: Cannot read property 'bind' of undefined"

Reply:
Please see PR #717

Reply:
Changes have been done, please review.